pom.xml

```diff
@@ -1277,7 +1277,7 @@
               <exclude>**/.settings/**/*</exclude>
 
               <!-- Proto/grpc generated wrappers -->
-              <exclude>**/sdks/python/apache_beam/runners/api/*.py</exclude>
+              <exclude>**/apache_beam/runners/api/*_pb2*.py</exclude>
```

Comment:
A comment that is slightly out of scope for this PR:

Seems like we have following problematic dependency chain now.

transforms/core.py -> runners/api/beam_runner_api_pb2.py -> runners/init.py -> all available implementations of PipelineRunner.

I was running into circular dependency issues in an unrelated PR simply because DirectRunner tried to use a class in transforms/core.py.

What is the best way to fix this ? Move runners/api to core/runners/api ? (probably we'll run into the same issue even if we automatically generate these files according to this PR).